Assets/ConnectApp/Utils/HttpManager.cs

```diff
@@ -205,9 +205,34 @@ public static class HttpManager {
                 StoreProvider.store.dispatcher.dispatch(new ScanEnabledAction {
                     scanEnabled = scan
                 });
+
+                DateTime endTime;
```

Comment:
use very similar codes in both the Try and Catch section, can we make it a function and call ?